freqtrade/edge/__init__.py

```diff
@@ -57,7 +57,9 @@ def __init__(self, config: Dict[str, Any], exchange, strategy) -> None:
         if self.config['stake_amount'] != constants.UNLIMITED_STAKE_AMOUNT:
             raise OperationalException('Edge works only with unlimited stake amount')
 
-        self._capital_percentage: float = self.edge_config.get('capital_available_percentage')
+        # Deprecated capital_available_percentage. Will use tradable_balance_ratio in the future.
+        self._capital_percentage: float = self.edge_config.get(
+            'capital_available_percentage', self.config['tradable_balance_ratio'])
```

Comment:
Once `capital_available_percentage` is removed completely, we should refactor edge to rely on the calculation done in `freqtradebot.py`.